### PR TITLE
NEON implementation of packed BabyBear

### PR DESCRIPTION
**Pull request by nbgl**

### CODE REVIEW

**Comment by nbgl** [comment]
i moved the contents of `baby-bear/src/lib.rs` to `baby-bear/src/baby_bear.rs`. unfortunately, the changelog doesn't seem to understand that.

File: baby-bear/src/aarch64_neon.rs [hunk 1]
```diff
@@ -0,0 +1,1071 @@
...
+    #[must_use]
+    const fn broadcast(value: BabyBear) -> Self {
+        Self([value; WIDTH])
+    }
+}
+
+impl Add for PackedBabyBearNeon {
+    type Output = Self;
+    #[inline]
+    #[must_use]
```
Comment (dlubarov) [inline baby-bear/src/aarch64_neon.rs:63]: nit: don't think these have an effect in this context, the docs say

> When used on a function in a trait implementation, the attribute does nothing.

**Review by dlubarov — APPROVED** [review]
LGTM

File: baby-bear/src/aarch64_neon.rs (hunk 1 above)
Comment (nbgl) [reply baby-bear/src/aarch64_neon.rs:63]: oh, oops!! i'll remove them

**Pull request merged**